README.md

```diff
@@ -21,31 +21,27 @@ End-to-End encryption with SNI proxy on top of a TCP multiplexer
 
 ## Fernet token
 
-The session master create a fernet token from client's config (aes/whitelist) and attach the hostname and a utc timestamp until this token is valid.
+The session master creates a Fernet token from the client's configuration (AES/whitelist) and attaches the hostname and a UTC timestamp until which the token is valid.
 
-```json
 {
-  "valid": 1923841,
-  "hostname": "myname.ui.nabu.casa",
-  "aes_key": "hexstring",
-  "aes_iv": "hexstring"
+"valid": 1923841,
+"hostname": "myname.ui.nabu.casa",
+"aes_key": "hexstring",
+"aes_iv": "hexstring"
 }
-```
-
-The SniTun server need to be able to decrypt this token to validate the client plausibility. SniTun initialize after that a challenge response handling to validate the AES key and make sure that it's the same client as they requests the fernet token from session master.
 
-SniTun server doesn't perform any user authentications!
+The SniTun server must be able to decrypt this token to validate the client's authenticity. SniTun then initiates a challenge-response handling to validate the AES key and ensure that it is the same client that requested the Fernet token from the session master.
 
-### Challenge/Response
+Note: SniTun server does not perform any user authentication!
 
-SniTun server create a SHA256 from a random 40bit value. They will be encrypted and send to client. This decrypt the value and perform again a SHA256 with this value and send it encrypted back to SniTun. If they is valid, he going into Multiplexer modus.
+Challenge/Response:
```

Comment:
This header style won't render as a header in markdown. Why do we not keep the existing header style?